source/includes/example-connect.rst

```diff
@@ -50,3 +50,24 @@ following command on one line:
                        clusterTwo02.fancyCorp.com:20020,
                        clusterTwo03.fancyCorp.com:20020
 
+To use ``mongosync`` with Atlas clusters, you add the
+:urioption:`ssl=true <ssl>` option to enable SSL. For example:
+
+.. code-block:: shell
+
+   mongosync \
+      --cluster0 'mongodb://clusterAdmin:superSecret@clusterOne01.fancyCorp.com:20020,clusterOne02.fancyCorp.com:20020,clusterOne03.fancyCorp.com:20020/?ssl=true' \
+      --cluster1 'mongodb://clusterAdmin:superSecret@clusterTwo01.fancyCorp.com:20020,clusterTwo02.fancyCorp.com:20020,clusterTwo03.fancyCorp.com:20020/?ssl=true'
+
+You can also use ``mongodb+srv`` connection strings with ``mongosync``.
+You do not need to add the :urioption:`ssl=true <ssl>` option to a
+``mongodb+srv`` connection string. For example:
+
+.. code-block:: shell
+
+   mongosync \
+      --cluster0 'mongodb+srv://clusterAdmin:superSecret@clusterOne01.fancyCorp.com:20020/' \
+      --cluster1 'mongodb+srv://clusterAdmin:superSecret@clusterTwo01.fancyCorp.com:20020/'
```

Comment:
These should not contain port numbers -- this command should be:

```
   mongosync \
      --cluster0 'mongodb+srv://clusterAdmin:superSecret@clusterOne01.fancyCorp.com/' \
      --cluster1 'mongodb+srv://clusterAdmin:superSecret@clusterTwo01.fancyCorp.com/'
```

^ tested that this command works as expected